plat/kvm/arm/setup.c

```diff
@@ -163,7 +167,14 @@ void __no_pauth _ukplat_entry(struct ukplat_bootinfo *bi)
 
 	fdt = (void *)bi->dtb;
 
+	/* With ukrsi, we defer console init after paging initialization */
```

Comment:
This comment as is does not add any value as it describes what is exactly below, similarly to:
```c
/* Print hello */
printf("Hello\n");
```
Useful comments explain why we do a behavior that is not immediately obvious to the reader of that code. Please update.